client/app/components/packages/landuse-form/lead-agency.js

```diff
@@ -1,8 +1,9 @@
 import Component from '@glimmer/component';
+import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 
 export default class PackagesLanduseFormLeadAgencyComponent extends Component {
-  @tracked chosenAccountName;
+  @tracked chosenAccountName = this.args.landuseForm.data.leadAgency ? this.args.landuseForm.data.leadAgency.name : null;
```

Comment:
Sweettt -- yes much better way than the placeholder!

Reply:
oh true!! I'll remove that. Don't know why I didn't think of that